onnxruntime/core/providers/webgpu/nn/fuse_utils.cc

```diff
@@ -65,7 +65,7 @@ std::string GetActivationSnippet(const Activation& activation, std::string value
     case ActivationKind::Clip:
       return "value = clamp(value, " + value_type_cast(activation.activation_params_.Clip.minimum_) + ", " + value_type_cast(activation.activation_params_.Clip.maximum_) + ");";
     case ActivationKind::HardSigmoid:
-      return "value = clamp(" + value_type_cast(activation.activation_params_.HardSigmoid.alpha_) + " * value + " + value_type_cast(activation.activation_params_.HardSigmoid.beta_) + ", 0.0" + ", 1.0" + ");";
+      return "value = clamp(" + value_type_cast(activation.activation_params_.HardSigmoid.alpha_) + " * value + " + value_type_cast(activation.activation_params_.HardSigmoid.beta_) + ", " + value_type_cast(0.0) + ", " + value_type_cast(1.0) + ");";
```

Comment:
>    return "value = clamp(" + value_type_cast(activation.activation_params_.HardSigmoid.alpha_) + " * value + " + value_type_cast(activation.activation_params_.HardSigmoid.beta_) + ", " + value_type_cast(0.0) + ", " + value_type_cast(1.0) + ");"; [](http://example.com/codeflow?start=2&length=246)

These lines must be wrapped to observe the limit of 120.

Reply:
can do but no lint is complaining :)